client/src/modules/reports/generate/budget_report/budget_report.config.js

```diff
@@ -31,13 +33,31 @@ function BudgetReportController($sce, Notify, SavedReports, AppCache, reportData
     vm.reportDetails = angular.copy(report);
   };
 
+  vm.onChangeIncludeSection = value => {
+    if (value === 0) {
+      vm.reportDetails.cashboxesIds = [];
+    }
+
+    vm.include_summary_section = value;
+  };
+
+  vm.onSelectCashboxes = (cashboxesIds) => {
+    vm.reportDetails.cashboxesIds = cashboxesIds;
+  };
+
   vm.numberYears = [
     { id : 1 }, { id : 2 }, { id : 3 }, { id : 4 }, { id : 5 },
   ];
 
   vm.preview = function preview(form) {
     if (form.$invalid) { return null; }
 
+    if (vm.include_summary_section === 0) {
+      vm.reportDetails.cashboxesIds = [];
+    }
+
+    vm.reportDetails.include_summary_section = vm.include_summary_section;
```

Comment:
**This is a hack** for reasons I couldn't figure out when I was using vm.reportDetails.include_summary_section the default should be false but I wasn't getting the result I was hoping for which is why I went with this solution

Reply:
> You probably just needed to clear your browser cache, since the report loads the browser cache by default (line 22). Not a big deal, just not very clean code.

You are right, it works now.